include/coap3/coap_session.h

```diff
@@ -24,6 +24,8 @@
  * @{
  */
 
+#include <stdbool.h>
```

Comment:
For historical reasons that I cant recall, it was decided that there should not be the use of boolean in the libcoap code, so this should not be used.